README.md

```diff
@@ -30,95 +30,94 @@ A Safe account with three owners and threshold equal three will be used as the s
 import { ethers } from 'ethers'
 import EthersSafe, { SafeTransaction } from 'safe-core-sdk'
 
-const provider = ethers.getDefaultProvider('homestead')
-const wallet1 = ethers.Wallet.createRandom().connect(provider)
-const wallet2 = ethers.Wallet.createRandom().connect(provider)
-const wallet3 = ethers.Wallet.createRandom().connect(provider)
+const web3Provider = // ...
+const provider = new ethers.providers.Web3Provider(web3Provider)
+const signer1 = provider.getSigner(0)
+const signer2 = provider.getSigner(1)
+const signer3 = provider.getSigner(2)
 
 // Existing Safe address (e.g. Safe created via https://app.gnosis-safe.io)
-// Where wallet1.address, wallet2.address and wallet3.address are the Safe owners
+// Where signer1, signer2 and signer3 are the Safe owners
 const safeAddress = "0x<safe_address>"
-const safeNonce = <safe_nonce>
 ```
 
-Create an instance of the Safe Core SDK with wallet1 connected as the signer.
+Create an instance of the Safe Core SDK with `signer1` connected as the signer.
 
 ```js
-const safeSdk = new EthersSafe(ethers, safeAddress, wallet1)
+const safeSdk = await EthersSafe.create(ethers, safeAddress, signer1)
 ```
 
 ### 1. Create a Safe transaction
 
 ```js
-const tx = new SafeTransaction({
+const tx = await safeSdk.createTransaction({
   to: safeAddress,
   value: '0',
   data: '0x',
-  nonce: safeNonce
```

Comment:
Did you mean this PR when we talked about allowing to pass a custom nonce? https://github.com/gnosis/safe-core-sdk/pull/19#discussion_r619105464

if yes, how was it implemented?

Reply:
Oh, I see `SafeTransactionDataPartial`. I could find in docs that you can pass custom nonce, though. Do you plan to add it?